src/sdk-reference/cpp/1/protocol/unregister-sub/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: sdk.html.hbs
+title: unregisterSub
+description: Used when unsubscribing to store a listener.
+---
+
+# registerSub
+
+Used when unsubscribing to remove a listener. 
```

Comment:
Not sure I got that right, but in any case, the description must be changed.
Should you accept the following suggestion, then you need to apply it to the page description too.

```suggestion
Removes a subscription's notifications listener.
```